readetexts-info.patch

```diff
@@ -1,39 +1,27 @@
-From e17694ec338ec1db8bf69f2e3c8bce613f5f7bad Mon Sep 17 00:00:00 2001
-From: Martin Abente Lahaye <tch@sugarlabs.org>
-Date: Tue, 17 Dec 2019 19:09:30 -0300
-Subject: [PATCH 1/3] info
-
----
- activity/activity.info | 13 ++++++++++---
- 1 file changed, 10 insertions(+), 3 deletions(-)
-
 diff --git a/activity/activity.info b/activity/activity.info
-index e1bf5be..4a77147 100644
+index c30b524..5eb8803 100644
 --- a/activity/activity.info
 +++ b/activity/activity.info
 @@ -1,12 +1,19 @@
  [Activity]
  name = Read ETexts
 -bundle_id = org.laptop.sugar.ReadEtextsActivity
 +bundle_id = org.sugarlabs.ReadETexts
- license = GPLv2+;LGPLv2+
 +metadata_license = CC0-1.0
++release_date = 2020-01-07
+ license = GPLv2+;LGPLv2+
  icon = read-etexts
- exec = sugar-activity ReadEtextsActivity.ReadEtextsActivity
+ exec = sugar-activity3 ReadEtextsActivity.ReadEtextsActivity
  show_launcher = yes
  activity_version = 27
-+release_date = 2019-06-13
  mime_types = text/plain;application/zip;application/rtf
 -summary = Download and read thousands of free e-books in plain text format from Project Gutenberg!
 -tags = Language;Tools
 +summary = Read thousands of free e-books
-+description = Read ETexts lets you read all the free e-texts from Project Gutenberg, or use Text to Speech to have them read to you. The built in offline catalog search lets you browse through and download over 30,000 great titles.
++description = Read ETexts lets you read all the free e-texts from Project Gutenberg, or use Text to Speech to have them read to you. The built in offline catalog search lets you browse through and download over 30,000 great titles. format from Project Gutenberg!
```

Comment:
That last `format from Project Gutenberg!` seems like a typo.

Reply:
I will remove that